clusterloader2/testing/batch/config.yaml

```diff
@@ -1,6 +1,6 @@
 {{$MODE := DefaultParam .MODE "Indexed"}}
 {{$NODES_PER_NAMESPACE := MinInt .Nodes (DefaultParam .NODES_PER_NAMESPACE 100)}}
-{{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 30}}
+{{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 100}}
```

Comment:
Revert this.

SIG-scalability currently only tests 30 pods per node by default.